internal/validation/duplicates.go

```
@@ -0,0 +1,237 @@
+		removeDuplicateTransactions(conn, chainId, duplicateTransactions)
+	}
+
+	duplicateLogs, err := findDuplicateLogsInRange(conn, chainId, startBlock, endBlock)
+	if err != nil {
+		return err
+	}
+	if len(duplicateLogs) == 0 {
+		log.Debug().Msg("No duplicate logs found in range")
+	} else {
+		log.Debug().Msgf("Found %d duplicate logs in range %v-%v: %v", len(duplicateLogs), startBlock, endBlock, duplicateLogs)
+		removeDuplicateLogs(conn, chainId, duplicateLogs)
```

golangci-lint (1.64.8)

Error return value is not checked

(errcheck)